test/unit/utils/to-gavel-result.test.js

```diff
@@ -0,0 +1,87 @@
+const { expect } = require('chai');
+const toGavelResult = require('../../../lib/utils/to-gavel-result');
+
+describe('toGavelResult', () => {
+  // Currently TV4 output is coerced to Amanda format.
+  // Then Amanda format is coerced to Gavel public API.
+  describe('given Amanda errors', () => {
+    let coercedErrors;
+
+    before(() => {
+      coercedErrors = toGavelResult({
+        length: 2,
+        0: {
+          property: ['users', 'username'],
+          propertyValue: 123,
+          attributeName: 'type',
+          attributeValue: 'string',
+          message: 'Amanda error message'
+        },
+        1: {
+          property: ['friends[2]', 'online'],
```

Comment:
Amanda produces `prop[n]` property, which appears to be invalid JSON pointer.

I've refactored the coercing logic so it produces `['fiends', 2, 'online']` JSON pointer format.

Reply:
This would be considered a breaking change, but it's rather a bug fix, because it fixes an unexpected property format. It's expected to be a list of stings, and it was resolved to a plain string sometimes.